### ISSUE TITLE
Suggestion: Iterating over a ListTemplate - add an each() method

### ISSUE THREAD

**Issue by dswitzer** [amcharts/amcharts4#34]
I have the need to frequently iterate over a ListTemplate (likes series). I noticed a ```ListTemplate``` has an ```iterator()``` function, but the usage seems overly complex. The examples of using the function look like this:

```
am4core.iter.each(dateAxis.renderer.labels.iterator(), function (label) {
    label.fillOpacity = label.fillOpacity;
});
```

Why not just give the ```ListTemplate``` an ```each()``` method, so that code could be changes to:

```
dateAxis.renderer.labels.each(function (label){
    label.fillOpacity = label.fillOpacity;
});
```

That would seem to save a good bit of code.

I could use ```forEach``` on the ```values``` array, but it seems like using the ```values``` property directly is discouraged.

**Comment by dswitzer** [comment]
A normal loop certainly works, but it's certainly way more verbose. I'm probably not alone in this, but I find this much more readable:

```
dateAxis.renderer.labels.each(function (label){
    label.fillOpacity = label.fillOpacity;
});
```

Less code, so less thinking about what's going on.

**Comment by martynasma** [comment]
I agree. I'll bring this up for discussion.

**Comment by Pauan** [comment]
As another alternative, what if we implemented the ES6 Iterator protocol for it?

```js
for (let label of dateAxis.renderer.labels) {
    label.fillOpacity = label.fillOpacity;
}
```

**Comment by dswitzer** [comment]
We need IE9 support, so I can't use ES6 syntax. I could run through a transpiler to convert it, but it would seem the iterator would simplify things (and it would simplify examples using the interator() function—which generates a callback function.)

**Comment by martynasma** [comment]
Implemented in [4.0.0-beta.25](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.25)
